SportHub/src/components/ArticleMenu/helpers/articleCanBeSaved.js

```diff
@@ -1,12 +1,12 @@
-const articleCanBeSaved = (tabData, selectedImage, alt) => {
+const articleCanBeSaved = (tabData, selectedImage, alt, article) => {
     let filled = true
     for (let i = 0; i < tabData.length; i++) {
```

Comment:
For the future: const isEmpty = Object.values(object).every(x => x === null || x === '');